backend/danswer/main.py

```diff
@@ -35,6 +35,20 @@ def validation_exception_handler(
     return JSONResponse(content=content, status_code=422)
 
 
+def value_error_handler(_: Request, exc: ValueError) -> JSONResponse:
+    return JSONResponse(
+        status_code=400,
+        content={"message": str(exc)},
+    )
+
+
+def general_exception_handler(_: Request, exc: Exception) -> JSONResponse:
+    return JSONResponse(
+        status_code=500,
+        content={"message": str(exc)},
```

Comment:
Seems a little dangerous. The Exception message could contain some information the user is not supposed to have access to?